drivers/flash/flash_stm32f2x.c

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2021 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <string.h>
+#include <drivers/flash.h>
+#include <init.h>
+#include <soc.h>
+
+#include "flash_stm32.h"
+
+#define STM32F2X_SECTOR_MASK		((uint32_t) 0xFFFFFF07)
```

Comment:
This shouldn't be needed, cf https://github.com/zephyrproject-rtos/zephyr/pull/36144